lblod/ember-rdfa-editor-lblod-plugins
# allow user to select how they want the article titles

### Overview
Made the user able to select the old article style without a only article title, and always full length articles

##### connected issues and PRs:
GN-5235


### Setup
None

### How to test/reproduce
Change the parameters in besluit config to:
```js
structure: {
        fullLenghtArticles: true,
        disableOnlyArticleSpecialName: true,
},
```
And observe how the style on the article titles change

### Challenges/uncertainties
None



### Checks PR readiness
